backend/model/db.js

```diff
@@ -13,6 +13,8 @@ const TitleModel = require('./titles')
 const TagModel = require('./tags')
 const UniversityModel = require('./universities')
 const DepartmentModel = require('./departments')
+const NotificationModel = require('./notification')
```

Comment:
all other models have plural file names, should be consistent

Reply:
that's not correct, follow does not have plural name,
and I dont think that this is an important feature or consistency that we need to spend time on

Reply:
yeah sure, @FarukOzderim of course they are not important, but they just occured to me and wanted to say it. and since for others you will edit the code anyway, I thought you could spare 5 seconds to also resolve this.

Reply:
if follow does not have plural name, then it also should have.

Reply:
@EnesToptas sure teammate, if you wish it, not a big problem :), changing the file names and the code now, if this makes you happy than all is well :)

Reply:
haha sorry if it came out aggresively, didn't mean it. good job! keep it up